src/ngraph_builder.cc

```diff
@@ -1293,6 +1350,8 @@ static Status TranslateConv2DBackpropInputOp(
           ng_padding_below, ng_padding_above,
           ng::Strides(ng_batch_shape.size() - 2, 1));
 
+  SaveNgOp(first_ng_op_map, op->name(), ng_data);
```

Comment:
Q: Did we not save the entry nodes here: https://github.com/NervanaSystems/ngraph-tf/pull/413/files?utf8=%E2%9C%93&diff=unified&w=1#diff-148e2a1788305495ad1a483e17b53eaaR1336

Reply:
Yes, correct.